physx/source/physxvehicle/src/physxmetadata/include/PxVehicleMetaDataObjects.h

```diff
@@ -76,15 +76,15 @@ struct MTorqueCurveProperty : public PxFixedSizeLookupTablePropertyInfo<PxVehicl
 	PX_PHYSX_CORE_API MTorqueCurveProperty();
 };
 
-#if PX_CLANG
+#if PX_LINUX && PX_CLANG
```

Comment:
Should this check be `!PX_ANDROID`, since this code may be applicable for other supported clang platforms(MacOS and Switch)?

Reply:
All the other places in the code base doing `ignored "-Wreserved-identifier"` are using `#if PX_LINUX && PX_CLANG`, this was the only one not to, so for consistency I followed the same pattern. And all the platforms built correctly like this.